src/batou_scm/tests/test_buildout.py

```diff
@@ -66,7 +60,6 @@ def test_no_dist_sources_configured_does_not_break(root):
     root.component += Buildout(config=File('buildout.cfg', content=''))
     root.component.configure()
     buildout = root.component._
-    assert 'develop +=' not in buildout.overrides.content
```

Comment:
Why is this assert no longer true? Removing it makes the whole test vage.

Reply:
No idea how that happened. I point to 2to3 ... :)